cmd/cape/cmd/token.go

```diff
@@ -41,9 +42,176 @@ var tokenCmd = &cobra.Command{
 	},
 }
 
+type createTokenReq struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+}
+
+type tokenRef struct {
+	Name        string    `json:"name"`
+	Description string    `json:"description"`
+	CreatedAt   time.Time `json:"created_at"`
+}
+
+type listTokensResponse struct {
+	Tokens []tokenRef `json:"tokens"`
+}
+
+type createTokenResponse struct {
+	Token string `json:"token"`
+}
+
+var listTokensCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List your account tokens",
+	Long: `List the tokens from your account.
+
+NOTE -- this will not list function tokens (ones made with cape token <function_id>).
+This will be fixed in the future.`,
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		verbose, err := cmd.Flags().GetBool("verbose")
+		if err != nil {
+			return err
+		}
+
+		url := C.EnclaveHost
+		req, err := http.NewRequest(http.MethodGet, url+"/v1/tokens", nil)
+		if err != nil {
+			return err
+		}
+
+		authToken, err := getAuthToken()
+		if err != nil {
+			return err
+		}
+
+		var bearer = "Bearer " + authToken
+		req.Header.Add("Authorization", bearer)
+
+		resp, err := http.DefaultClient.Do(req)
+		if err != nil {
+			return err
+		}
+
+		if resp.StatusCode != http.StatusOK {
+			cmd.SilenceUsage = true
+			if verbose {
+				fmt.Println("received status code", resp.StatusCode)
+			}
+
+			return fmt.Errorf("something went wrong")
```

Comment:
We should have bit more info than this no? Or are we not returning anything yet?

Reply:
was just concerned that the server doesn't sanitize errors yet so it would show pg errors etc

Reply:
Gotcha! All good.